ext/nnstreamer/tensor_filter/tensor_filter_tensorflow_lite.c

```diff
@@ -32,16 +32,6 @@
 #include "tensor_filter_tensorflow_lite_core.h"
 #include "tensor_common.h"
 
-/**
- * @brief nnapi hw type string
- */
-static const char *nnapi_hw_string[] = {
```

Comment:
how about moving this string list into _lite_core.cc and make function tflite_get_nnapi_hw_type() instead the static function in _lite.c